javascript/javascript_sdk/src2/get_data.js

```diff
@@ -0,0 +1,95 @@
+const google_protobuf_timestamp_pb = require('google-protobuf/google/protobuf/timestamp_pb.js');
+const sensr_client = require('./sensr_client');
+
+const get_zone_data = (network) => {
+    let client = sensr_client.SensrClient(network);
+    client.listenToObjectUpdate(response => {
```

Comment:
This is a just question, 
How this function recognize the lamda is for streamCallback or eventCallback or timestampCallback?

Reply:
I think I didn't use them properly in my code I will fix them. I can use null for callback that I don't use. For instance, 
```
const get_zone_data = (client) => {
    client.listenToObjectUpdate(null, null, response => {
      let zone_lists = response.getZonesList();
      zone_lists.forEach(zone_list => {
      ```